bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/views/report/list/ReportListView.java

```diff
@@ -48,17 +60,20 @@
     private String currentFilename;
     private IProject currentProject;
     private ShowFilterConfigurationDialog showfilterconfigurationdialog;
-
+    private Label message;
+    private Composite client;
     public ReportListView() {
     }
 
     public void createPartControl(Composite parent) {
 
         this.parent = parent;
 
```

Comment:
Added a client area to place widgets to. There is a new message Label to display the project name